frontend-vite/src/components/Avatar.jsx

```diff
@@ -1,22 +1,12 @@
 import React from "react";
 
-const  Avatar = ({ src, alt, size }) => {
-
-  // let avatarSize;
-  // if(size == "small"){
-  //   avatarSize = 20;
-  // } else if(size == "medium"){
-  //   avatarSize = 30;
-  // } else {
-  //   avatarSize = 40;
-  // };
-    return (
-      <img
-        className={`inline-block w-12 h-12 rounded-full border-2 border-blue-400 overflow-hidden`}
-        src={src}
-        alt={alt}
-      />
-    );
-  }
-  
-  export default Avatar
\ No newline at end of file
+const Avatar = ({ src, alt, size }) => {
+  return (
+    <img
+      className={`inline w-${size} h-${size} rounded-full border-2 border-blue-400 overflow-hidden`}
```

Comment:
do you not need [ ] anymore for size?

Reply:
No, the [] are for arbitrary sizes if we want to break out of the styling system.